wordpress-mobile/WordPress-Android
# Implement click logic on site items

Fixes #13347 

This PR adds logic to list items in the My Site to click on items and a lot of tests. There is almost no logic in place so it should all be fairly straightforward. The only thing that contains logic is clicking on stats where (depending on the site) it opens either stats, login or jetpack install flow.

To test:
- Turn on the my site feature flag and restart the app
- Try clicking on all the items
- Check that the correct screen is opened on click 

PR submission checklist:
